### ISSUE TITLE
No way to impiment?

### ISSUE THREAD

**Issue by SoLongSidekick** [MikesMachines/MotorControl#1]
I'm sorry if I'm missing something, but there's no code or programs here it's just images of hardware wiring. How would I go about coding the ESP32 or acquiring the Android app?

Also, why do the 3 and 4 channel examples shown each have 2 and 3 outputs respectively? If using a PWM controller like the one linked at bottom do I just wire the +V and GND directly into the inputs of a MOSFET board?

https://www.aliexpress.us/item/3256806543996830.html?spm=a2g0o.order_list.order_list_main.951.2d1f194dbNrKHd&gatewayAdapt=glo2usa

**Comment by MikesMachines** [comment]
So those boards are just some of the examples used in different machines. 

Those specific 400w mosfets, have their positive plane connected. If you screw into the pos, the other side is on - connected or not. 

So the power switch is positive, and pos lines run straight to the motor. The neg line goes from the mosfet to the motor. This is a board specific issue. There are many options. 

I usually share a ground for all the mosfets.

**Comment by MikesMachines** [comment]
I think I might understand better now about the inputs not matching. One of the uses a GX16 aviation connector with a 6 core wire terminal. 

The 4 channel one of the right has DC  barrel jack outputs underneath each mosfet. Hope I helped ;)

**Comment by breadfan69** [comment]
What a fantastic project, but where is the code and app?

**Comment by alcomys** [comment]
> What a fantastic project, but where is the code and app?

Yes that's the question, I think this design does not belong on GitHub because it has nothing to do with programming and embedded IOT. 
This is just an guide to build something that not exist. 

Than the owner shows pictures of a app running some? and don't share any scripts and or the app self. 
So this mean that you have to make all your self including the app.
